scripts/generateDepcheckrcJaegerUI.js

```diff
@@ -27,7 +27,7 @@ const packageNames = [
   ...babelConfiguration.plugins,
 ];
 
-const otherPackages = ['jest-environment-jsdom'];
+const otherPackages = ['jest-environment-jsdom', 'babel-plugin-react-remove-properties'];
```

Comment:
wait, why is it not auto-discovered from babelrc? I don't see babelrc change, isn't it necessary to make babel actually use the plugin, not just install it as dep?

Reply:
We are not using Babel plugins for production builds.

I was also confused earlier, since we have a `babel-transform.js` file. I tried adding the plugin to that file, but it didn't worked.
Later realised that it is only being used for tests, and not for builds.

For builds, we are using `vite.config.js` and depcheck can't find that out that we are using babel plugin inside vite.config.js

Reply:
Parsing it seems tricky. I applied another workaround, since depcheck by default supports .babelrc files.